source/features/preserve-file-finder-term.tsx

```diff
@@ -0,0 +1,44 @@
+import features from '../libs/features';
+import select from 'select-dom';
+import elementReady from 'element-ready';
+
+const unloadHandler = (): void => {
+	const inputElement = select<HTMLInputElement>('.js-tree-finder > .breadcrumb > #tree-finder-field');
+	if (inputElement) {
+		sessionStorage.setItem('rgh-file-finder-term', inputElement.value);
+	}
+};
+
+// Set the input field value & trigger event
+const setValueInField = async (): Promise<void> => {
+	const value: string = sessionStorage.getItem('rgh-file-finder-term') ?? '';
```

Comment:
`??` isn’t needed, let it be `undefined`, we’re checking for it in the condition anyway. The explicit type isn’t required here either.